libs/mimir/src/adapters/primary/common/dsl.rs

```diff
@@ -246,7 +252,7 @@ fn build_matching_condition(q: &str, query_type: QueryType) -> serde_json::Value
         //     Caisse Primaire d'Assurance Maladie de Haute Garonne, 33 Rue du Lot, 31100 Toulouse
         QueryType::FUZZY => json!({
             "match": {
-                "label": {
+                "full_label.ngram": {
                     "query": q,
                     "minimum_should_match": "1<-1 3<-2 9<-4 20<25"
```

Comment:
Not really related, but
```suggestion
                    "minimum_should_match": "1<-1 3<-2 9<-4 20<25%"
```
I think this is a 25% and not 25. A 25 does not really make sense when I read [the doc](https://www.elastic.co/guide/en/elasticsearch/reference/7.16/query-dsl-minimum-should-match.html).
And there is a subtle f64 [in bragi2](https://github.com/CanalTP/mimirsbrunn/blob/da72c2fac5887d3c6fbbc2b4d69201ecb9a95319/libs/bragi/src/query.rs#L373)

Reply:
thank you, indeed it was a nonsense